user/languages/python.md

```diff
@@ -170,6 +170,9 @@ jobs:
   include:
     - name: "Python 3.8.0 on Xenial Linux"
       python: 3.8           # this works for Linux but is ignored on macOS or Windows
+    - name: "Python 3.6.10 on FreeBSD"
```

Comment:
The current default python version on FreeBSD is 3.7, maybe we want to rebuild the image and update this in the future?